service/service.go

```diff
@@ -21,11 +21,21 @@ import (
 	"errors"
 
 	pb "github.com/kubeflow/metadata/api"
+	mlmd "github.com/kubeflow/metadata/ml_metadata"
```

Comment:
This is weird to me. These protos are third party, and we're building and using them under our package name, which seems wrong. How do we keep the ml_metadata package name (and not use ours here)?

Reply:
The goal of the package is to compile a Go client for MLMD. Maybe 
 changes the package path to github.com/kubeflow/metadata/`clients/ml_metadata` makes it clear.

WDYT?

Reply:
I guess I'm asking if we can use package name like `github.com/ml-metadata/ml_metadata/ml_metadata_store_proto` or something along those lines. Is this possible or not really when using third party protos in Go? 

If this is not possible from the client side (and I suspect it's not), then I think the right solution here could be to get ml-metadata to export the Go proto files. So that way, we can use consistent package names that works across both Bazel and Go. 

I will lgtm this to unblock you, but I think we can look into publishing those Go proto files upstream and rely on them instead once it's ready. Does this work for you?

Reply:
> If this is not possible from the client side (and I suspect it's not), then I think the right solution here could be to get ml-metadata to export the Go proto files.

I think that is the only way to define `github.com/ml-metadata/ml_metadata/ml_metadata_store_proto`: move this compiled Go package here to MLMD's github repo. If this happens, we can import it as other Go packages, which is cleaner solution.

I will land this PR for now. Thanks!